Create_Directories OneFS 8.2.2 API issue

Running the isilon_create_directories.py script on OneFS 8.2.2 gets interrupted with the following error:

[DEBUG] response body:
{
"errors" :
[

{
"code" : "AEC_NOT_FOUND",
"message" : "Path not found: /4/protocols/hdfs/settings."
}
]
}

[ERROR] Path not found: /4/protocols/hdfs/settings.

--- a/setup.py
+++ b/setup.py
@@ -33,6 +33,7 @@ setuptools.setup(
         'isi-sdk-8-1-1 ~= 0.2.7',
         'isi-sdk-8-2-0 ~= 0.2.7',
         'isi-sdk-8-2-1 ~= 0.2.7',
+        'isi-sdk-8-2-2 ~= 0.2.7',
         'requests >= 2.20.0',
         'setuptools >= 41.0.0',
     ],

--- a/src/isilon_hadoop_tools/onefs.py
+++ b/src/isilon_hadoop_tools/onefs.py
@@ -70,7 +70,8 @@ ONEFS_RELEASES = {
     '8.1.3.0': 0x80103500000000D,
     '8.2.0.0': 0x80200500000000B,
     '8.2.1.0': 0x802015000000004,
-    '8.2.2.0': 0x802025000000000,
+    '8.2.2.0': 0x802025000000007,
+    '8.2.3.0': 0x802035000000000,
 }
 
 
@@ -523,13 +524,16 @@ def sdk_for_revision(revision, strict=False):
     if ONEFS_RELEASES['8.2.1.0'] <= revision < ONEFS_RELEASES['8.2.2.0']:
         import isi_sdk_8_2_1
         return isi_sdk_8_2_1
+    if ONEFS_RELEASES['8.2.2.0'] <= revision < ONEFS_RELEASES['8.2.3.0']:
+        import isi_sdk_8_2_2
+        return isi_sdk_8_2_2
     # At this point, either the cluster is too old or too new;
     # however, new clusters still support old SDKs,
     # so, unless the caller asks to fail here, we'll fall back to the newest supported SDK.
     if strict:
         raise UnsupportedVersion('There is no SDK for OneFS revision {0}!'.format(hex(revision)))
-    import isi_sdk_8_2_1
-    return isi_sdk_8_2_1  # The latest SDK available.
+    import isi_sdk_8_2_2
+    return isi_sdk_8_2_2  # The latest SDK available.
 
 
 def accesses_onefs(func):

--- a/tests/test_onefs.py
+++ b/tests/test_onefs.py
@@ -22,6 +22,7 @@ import isi_sdk_8_1_0
 import isi_sdk_8_1_1
 import isi_sdk_8_2_0
 import isi_sdk_8_2_1
+import isi_sdk_8_2_2
 import pytest
 
 from isilon_hadoop_tools import IsilonHadoopToolError, onefs
@@ -68,7 +69,7 @@ def test_api_error_str(api_error):
 @pytest.mark.parametrize(
     'revision, expected_sdk',
     [
-        (0, isi_sdk_8_2_1),
+        (0, isi_sdk_8_2_2),
         (onefs.ONEFS_RELEASES['7.2.0.0'], isi_sdk_7_2),
         (onefs.ONEFS_RELEASES['8.0.0.0'], isi_sdk_8_0),
         (onefs.ONEFS_RELEASES['8.0.0.4'], isi_sdk_8_0),
@@ -79,8 +80,9 @@ def test_api_error_str(api_error):
         (onefs.ONEFS_RELEASES['8.1.2.0'], isi_sdk_8_1_1),
         (onefs.ONEFS_RELEASES['8.2.0.0'], isi_sdk_8_2_0),
         (onefs.ONEFS_RELEASES['8.2.1.0'], isi_sdk_8_2_1),
-        (onefs.ONEFS_RELEASES['8.2.2.0'], isi_sdk_8_2_1),
-        (float('inf'), isi_sdk_8_2_1),
+        (onefs.ONEFS_RELEASES['8.2.2.0'], isi_sdk_8_2_2),
+        (onefs.ONEFS_RELEASES['8.2.3.0'], isi_sdk_8_2_2),
+        (float('inf'), isi_sdk_8_2_2),
     ],
 )
 def test_sdk_for_revision(revision, expected_sdk):

